src/utils/getName.js

```diff
@@ -10,7 +10,7 @@ export default t => path => {
   let namedNode
 
   path.find(path => {
-    // const X = styled
+    // X = styled
```

Comment:
This was backwards see: https://astexplorer.net/#/gist/3e299bf92f4651d5230a94ef129e5509/latest